testsuite/smoke-test/src/operational_tooling.rs

```diff
@@ -1061,8 +1060,12 @@ async fn test_verify_validator_state() {
     let result = op_tool.verify_validator_state(&backend).unwrap();
     assert_eq!(result.in_validator_set, Some(true));
     assert_eq!(result.consensus_key_match, Some(false));
+    assert_eq!(result.consensus_key_unique, Some(true));
+    assert_eq!(result.validator_network_key_match, Some(true));
+    assert_eq!(result.fullnode_network_key_match, Some(true));
 
     // TODO(khiemngo): consider adding test where the validator is no longer in set
+    // TODO(khiemngo): consider adding test where consensus key is not unique
```

Comment:
One way of doing this would be to create a swarm of several nodes, and then manually change the consensus key of one of them to match another. This would probably requiring manually overwriting the consensus key in storage and then calling the `rotate_consensus_key` function of the operator tool 😄